crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -105,15 +139,37 @@ pub struct StandardMaterial {
 
     /// Support two-sided lighting by automatically flipping the normals for "back" faces
     /// within the PBR lighting shader.
-    /// Defaults to false.
+    ///
+    /// Defaults to `false`.
     /// This does not automatically configure backface culling, which can be done via
     /// `cull_mode`.
     pub double_sided: bool,
-    /// Whether to cull the "front", "back" or neither side of a mesh
-    /// defaults to `Face::Back`
+
+    /// Whether to cull the "front", "back" or neither side of a mesh.
+    ///
+    /// Defaults to `Face::Back`.
     pub cull_mode: Option<Face>,
+
+    /// Whether to shade this material.
+    ///
+    /// Normals, occlusion textures, roughness, metallic, reflectance and
+    /// emissive are ignored if this is set to `true`.
     pub unlit: bool,
+
+    /// How to interpret the alpha channel of the `base_color_texture`.
+    ///
+    /// By default, it's `Opaque`, therefore completely ignored.
+    /// Note that currently bevy handles poorly semi-transparent textures. You
+    /// are likely to encounter the following bugs:
+    ///
+    /// - When two `AlphaMode::Blend` materials occupy the same pixel, only one
+    ///   material's color will show.
```

Comment:
I don’t understand this point. It sounds like you’re saying that alpha blending does not work. Last I checked it does. If it doesn’t work that shouldn’t be in documentation it should just be fixed.

Reply:
Ah. I simply had poor results trying to use `Blend` mode, but I don't quite remember which one. I need to defer to someone who knows this better than myself :sweat_smile: